travis/run_pytest.bash

```diff
@@ -5,8 +5,8 @@
 
 set -euxo pipefail
 
-python -m pytest -v --capture=no --cov=cclib --cov-report=term --cov-report=html --terse test
+python -m pytest -v --traceback --capture=no --cov=cclib --cov-report=term --cov-report=html --terse test
```

Comment:
I would stick this at the end as an attempt to differentiate between `pytest` flags and cclib flags.